bridges/SteamBridge.php

```diff
@@ -56,26 +21,15 @@ class SteamBridge extends BridgeAbstract {
 
 	public function collectData(){
 
-		$username = $this->getInput('username');
-		$params = array(
-			'cc' => $this->getInput('currency')
-		);
-
-		$url = self::URI . 'wishlist/id/' . $username . '?' . http_build_query($params);
+		$userid = $this->getInput('userid');
 
-		$targetVariable = 'g_rgAppInfo';
+		$sourceUrl = self::URI . 'wishlist/profiles/' . $userid . '/wishlistdata?p=0';
 		$sort = array();
 
-		$html = '';
-		$html = getSimpleHTMLDOM($url)
-			or returnServerError("Could not request Steam Wishlist. Tried:\n - $url");
-
-		$jsContent = $html->find('.responsive_page_template_content script', 0)->innertext;
-
-		if(preg_match('/var ' . $targetVariable . ' = (.*?);/s', $jsContent, $matches)) {
-			$appsData = json_decode($matches[1]);
+		if(!$json = file_get_contents($sourceUrl)) {
```

Comment:
Use [`getContents`](https://github.com/RSS-Bridge/rss-bridge/wiki/The-getContents-function) for raw data. It is based on cURL and respects the RSS-Bridge configuration.

```suggestion
		if(!$json = getContents($sourceUrl)) {
```

Also, this statement doesn't require an if-statement if you write it like this:

```php
		$json = getContents($sourceUrl)) 
			or returnServerError('Could not get content from wishlistdata (' . $sourceUrl . ')');

		$appsData = json_decode($json);
```